### ISSUE TITLE
Someone says it's blocking

### ISSUE THREAD

**Issue by JDJGInc** [Helow19274/aiogTTS#2]
A friend named Soos#6008 on discord says it's somehow blocking.

**Comment by BeastImran** [comment]
I agree with this issue. aiogtts has async methods but are using blocking methods in them.

Please modify it to really make it asynchronous supported ................

**Comment by JDJGInc** [comment]
Anyway I actually made a test, and AioGTTs is only 125 MS max from web requests plus all the shengians I do.

**Comment by JDJGInc** [comment]
so it's a lot faster than Soos thought lol.

**Comment by BeastImran** [comment]
Cool 👍🏻

**Comment by JDJGInc** [comment]
It's way slower on large text btw, it may be blocking, no idea, the main problem is the write to bytes with large text.

**Comment by JDJGInc** [comment]
Okay, so my feeling is that it's actually bs4 that's blocking try wrapping it in asyncio.to_thread or executor :)
